<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login</class>
 <widget class="QMainWindow" name="login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>855</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>540</width>
    <height>855</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Nagato</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icons/eaxum_icon.png</normaloff>icons/eaxum_icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">*{ 
font: 8pt &quot;Consolas&quot;;;
font-size:24px;
}

#label_username{
background-color:transparent;
color: white;
}

#label_password{
background-color:transparent;
color: white;
}


#login_text{
background-color:transparent;
color: white;
}
#login_icon{
background-color: transparent;

}
#login{
background: url(:/img/img/bg_1.png) no-repeat center center fixed;
}

QLineEdit{
font-size:17px;
background:transparent;
border:none;
color: white;
border-bottom: 1px solid gray;
}

QFrame
{
background-color: #333;
border-radius: 25px;
	
}

QPushButton
{
	background-color: rgb(0, 3, 232);
	color: #fff;
	font-size: 20px;
	font-weight: bold;
	border: none;
	border-radius: 25px;
	padding: 5px;

}


QPushButton::disabled
{
	background-color: #5c5c5c;

}


QPushButton::pressed
{
	background-color: rgb(0, 3, 232)

}

QPushButton::hover
{
	background-color: rgb(56, 196, 255);
	

}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="maximumSize">
       <size>
        <width>361</width>
        <height>501</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="login_icon">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>30</y>
         <width>111</width>
         <height>111</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>icons/login.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="login_text">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>110</y>
         <width>141</width>
         <height>81</height>
        </rect>
       </property>
       <property name="text">
        <string>Login Here</string>
       </property>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>170</y>
         <width>221</width>
         <height>191</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLineEdit" name="host_url">
          <property name="inputMask">
           <string/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="maxLength">
           <number>32767</number>
          </property>
          <property name="placeholderText">
           <string>Host Url</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="username_input">
          <property name="inputMask">
           <string/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="maxLength">
           <number>32767</number>
          </property>
          <property name="placeholderText">
           <string>Username</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="password_input">
          <property name="inputMask">
           <string/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="maxLength">
           <number>32767</number>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
          <property name="placeholderText">
           <string>Password</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="login_button">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>390</y>
         <width>191</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
